internal/versions/versions.go

```diff
@@ -33,12 +33,12 @@ func (h *handler) ListComponentVersions(ctx context.Context, params operations.L
 	return operations.NewListComponentVersionsOK().WithPayload(
 		&models.ListVersions{
 			Versions: models.Versions{
-				"assisted-installer-service":                 h.versions.SelfVersion,
-				"image-builder":                              h.versions.ImageBuilder,
-				"discovery-agent":                            h.versions.AgentDockerImg,
-				"ignition-manifests-and-kubeconfig-generate": h.versions.KubeconfigGenerator,
-				"assisted-installer":                         h.versions.InstallerImage,
-				"assisted-installer-controller":              h.versions.ControllerImage,
+				"assisted-installer-service":    h.versions.SelfVersion,
+				"image-builder":                 h.versions.ImageBuilder,
+				"discovery-agent":               h.versions.AgentDockerImg,
+				"assisted-ignition-generator":   h.versions.KubeconfigGenerator,
```

Comment:
Consider renaming h.versions.KubeconfigGenerator, while you are at it

Reply:
fixed